api/app/main.py

```
@@ -17,7 +24,49 @@
     logging.config.dictConfig(config)
 
 
+# Configure global JSON encoder for datetime objects
+def setup_json_encoder():
+    """Configure FastAPI to use consistent datetime serialization"""
+    from fastapi.encoders import jsonable_encoder
+    from fastapi.responses import JSONResponse
```

Ruff (0.12.2)

Redefinition of unused `JSONResponse` from line 10

Remove definition: `JSONResponse`

(F811)